clusterloader2/cmd/clusterloader.go

```diff
@@ -39,7 +39,10 @@ var (
 func initClusterFlags() {
 	pflag.StringVar(&clusterLoaderConfig.ClusterConfig.KubeConfigPath, "kubeconfig", "", "Path to the kubeconfig file")
 	// TODO(krzysied): query actual number of nodes with client (if not set).
-	pflag.IntVar(&clusterLoaderConfig.ClusterConfig.Nodes, "nodes", 0, "number of nodes")
+	pflag.IntVar(&clusterLoaderConfig.ClusterConfig.Nodes, "nodes", 0, "Number of nodes")
+	pflag.StringVar(&clusterLoaderConfig.ClusterConfig.Provider, "provider", "", "Cluster provider")
```

Comment:
What is the usecase for provider?

I know that there are features that may be provider specific (e.g. storage), but does it really matter?